compact_test.go

```diff
@@ -1058,3 +1065,45 @@ func TestDeleteCompactionBlockAfterFailedReload(t *testing.T) {
 		})
 	}
 }
+
+func TestOpenBlockWithHook(t *testing.T) {
+	//init action
+	original := filepath.Join(string(filepath.Separator), "tmp", fmt.Sprintf("dev-%d", time.Now().Unix()))
+	mountpoint := filepath.Join(string(filepath.Separator), "tmp", fmt.Sprintf("mountpoint-%d", time.Now().Unix()))
+	//create block will be successful because hook server does not start
+	path, err := createBlock(t, original, genSeries(1, 1, 200, 300))
+	testutil.Ok(t, err)
+	_, file := filepath.Split(path)
+	server, err := testutil.NewServer(t, original, mountpoint, testutil.Hook(TestRenameHook{}))
```

Comment:
`testutil.NewServer` it is a bit unclear what is this new server. Best to have it as before `testutil.NewFuseServer` or maybe put as a child package of `testutil/fuse` so that this becomes `fuse.NewServer`

Reply:
Acceptd and Updated